[DX-696] Update field-based-permissions.md (#3276)
TykTechnologies/tyk-docs: tyk-docs/content/graphql/field-based-permissions.md

## Setup field based permissions in Dashboard

Restricted and allowed types and fields can also be set up via Tyk Dashboard.

1. *Optional:* Configure a Policy from **System Management > Policies > Add Policy**.
2. From **System Management > Keys > Add Key** select a policy or configure directly for the key.
3. Select your GraphQL API (marked as *GraphQL*).

> 4. Enable either **Block list** or **Allow list**. It's not possible to have both enabled at the same time.

Reviewer: Is the default behaviour to have *Blocked List* switch enabled with the *Allow List* switch hidden? 
If the *Block List* switch is disabled does this then display and enable the *Allow List* switch?

If so, maybe add an explanation of this in bullet point 4.

> By default all *Types* and *Fields* will be unchecked. By checking a *Type* or *Field* you will disallow to use it for any GraphQL operation associated with the key.

Reviewer: ```suggestion
### Block list

By default all *Types* and *Fields* will be unchecked. By checking a *Type* or *Field* you will disallow to use it for any GraphQL operation associated with the key.
```

Author: I completely agree with this change but I don't know why `### Block list` was missing in the first place. It's there in commit `6e4da30dcb878755cf60791c443abfb52b1a5714`, line 160. Weird

> For example, the settings from the screenshot below would block `code` and `countries` fields in `Continent` type as well as `latt` and `longt` fields in `Coordinates` type from usage.

Reviewer: ```suggestion
For example, the settings illustrated below would block the following:
- `code` and `countries` fields in `Continent` type.
- `latt` and `longt` fields in `Coordinates` type.
```

{{< img src="/img/dashboard/system-management/field-based-permissions-blocklist.png" alt="field-based-permissions" >}}

### Allow list

> By default all *Types* and *Fields* will be unchecked. By checking a *Type* or *Field* you will allow to use it for any GraphQL operation associated with the key.

Reviewer: ```suggestion
By default all *Types* and *Fields* will be unchecked. By checking a *Type* or *Field* you will allow it to be used for any GraphQL operation associated with the key.
```

> For example, the settings from the screenshot below would only allow `code` field in `Continent` type as well as `code` and `name` fields in `Language` to be used. Note that since no query is allowed (no field in `Query` type is checked), you will not be able to run any query whatsoever.

Reviewer: ```suggestion
For example, the settings illustrated below would only allow the following:
- `code` field in `Continent` type.
- `code` and `name` fields in `Language` type.

Note that the `Query` type is unchecked. Subsequently, you will not be able to run any query.
```
